06week/Checkpoint2.js

```diff
@@ -0,0 +1,45 @@
+
+// Given 1000 digits of PI as strings, return an array of the digits as numbers
+const nums = strNums.join();
+
+console.log(nums);
+
+// Find the sum of the even values
+//had to filter even numbers first
+const findEvens = strNums.filter(num => num % 2 ===0);
+//adding the total of the even values through reduce
+const sumEvens = findEvens.reduce((total, num) => {
+  return [(+total) + (+num)];
+});
+
+console.log(sumEvens);
```

Comment:
this returns an array with the sum as the first value, how would you just return the sum of findEvens? 
`const sumEvens = findEvens.reduce((total, num) => {
  return (+total) + (+num);
});`